controllers/monitor.go

```diff
@@ -0,0 +1,169 @@
+package controllers
+
+import (
+	"context"
+	monitoringv1 "github.com/prometheus-operator/prometheus-operator/pkg/apis/monitoring/v1"
+	trustyaiopendatahubiov1alpha1 "github.com/trustyai-explainability/trustyai-service-operator/api/v1alpha1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/log"
+)
+
+var isCentralServiceMonitorCreated = false
+
+// generateServiceMonitorSpecCentral generates the ServiceMonitor spec for central ServiceMonitor
+func generateServiceMonitorSpecCentral(deploymentNamespace string) *monitoringv1.ServiceMonitor {
+	serviceMonitor := &monitoringv1.ServiceMonitor{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      serviceMonitorName,
+			Namespace: deploymentNamespace,
+			Labels: map[string]string{
+				"modelmesh-service": "modelmesh-serving",
+			},
```

Comment:
Is this label used by some part of serving monitoring stack to filter?

Reply:
Good point, perhaps it's not needed.
Moved to 
- https://github.com/trustyai-explainability/trustyai-service-operator/issues/82